src/lib/foundation/Chooser.svelte

```diff
@@ -45,4 +52,15 @@
     .dropdownHidden {
         @apply hidden;
     }
+    .chooser-text{
+        text-align: left;
+        display: flex;
+    }
+    .close-button{
+        margin-left: auto;
+    }
+    .icon {
```

Comment:
Because we use Tailwind, I actually want to get away from using <style> blocks in our components. I consider this legacy.

`chooser-text` becomes `text-left flex`
`close-button` becomes `ml-auto`
`icon` becomes `w-[30px] h-[30px]`